### PR TITLE
Feature/implement comment

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
image
image

### CODE REVIEW

File: practice-two/src/styles/layouts/comment.scss [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    &-img {
+      max-width: toRem(19.19px);
+      min-height: toRem(19.19px);
+    }
+  }
+
+  &__name {
+    display: flex;
+    align-items: center;
```
Comment (vuphan-agilityio) [inline practice-two/src/styles/layouts/comment.scss:37]: @thanhtrantruong47 this two properties can reusable by using a placeholder in sass

File: practice-two/src/styles/layouts/comment.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  &__card {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: toRem(20px);
+  }
+
+  .comment__head {
+    display: flex;
+    justify-content: space-between;
```
Comment (vuphan-agilityio) [inline practice-two/src/styles/layouts/comment.scss:14]: @thanhtrantruong47 The same as the comment below

File: practice-two/src/styles/layouts/comment.scss [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      }
+    }
+
+    &__name {
+      font-size: toRem(20px);
+      padding: toRem(15px) 0 toRem(12px) 0;
+
+      &-img {
+        max-width: toRem(24px);
+        max-height: toRem(24px);
```
Comment (vuphan-agilityio) [inline practice-two/src/styles/layouts/comment.scss:80]: @thanhtrantruong47 Consider building a mixin in sass to set these two properties.

File: practice-two/src/index.html [hunk 4]
```diff
@@ -12,11 +12,11 @@
   <!-- sign up start  -->
   <div class="sign-up">
     <div class="container sign-up">
```
Comment (sinhnguyen-agilityio) [inline practice-two/src/index.html:14]: The parent of this element has class sign-up. Why you still using this class here?

File: practice-two/src/index.html [hunk 5]
```diff
@@ -12,11 +12,11 @@
...
-    <p class="sign-up__text">Sign up and get 20% off to your first order. <a class="sign-up__link"
-        href="javascript:void(0)">Sign Up
-        Now</a></p>
-    <a href="javascript:void(0)"><img class="sign-up__icon" src="assets/images/headers/off.svg" alt="off"></a>
-  </div>
+      <p class="sign-up__text">Sign up and get 20% off to your first order. <a class="sign-up__link"
```
Comment (sinhnguyen-agilityio) [inline practice-two/src/index.html:15]: Please check class name. It should be sign-up__content, sign-up__description or sign-up__advertise

File: practice-two/src/styles/modules/title.scss [hunk 6]
```diff
@@ -27,6 +31,10 @@
     &--center {
```
Comment (sinhnguyen-agilityio) [inline practice-two/src/styles/modules/title.scss:31]: What is different between the title-secondary--center and title-secondary--left?

File: practice-two/src/styles/utils/placeholder.scss [hunk 7]
```diff
@@ -0,0 +1,9 @@
+%space {
+  display: flex;
+  justify-content: space-between;
+}
+
+%center {
```
Comment (vuphan-agilityio) [inline practice-two/src/styles/utils/placeholder.scss:6]: @thanhtrantruong47 flexbox-items-centered

File: practice-two/src/styles/utils/placeholder.scss [hunk 8]
```diff
@@ -0,0 +1,9 @@
+%space {
```
Comment (vuphan-agilityio) [inline practice-two/src/styles/utils/placeholder.scss:1]: @thanhtrantruong47 flexbox-items-space-between

**Pull request merged**